fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/LaoDetailViewModel.java

```diff
@@ -463,35 +435,26 @@ public void createNewRollCall(
       return;
     }
     String channel = lao.getChannel();
-    CreateRollCall createRollCall;
-    String laoId = channel.substring(6); // removing /root/ prefix
-    createRollCall =
+    // FIXME Location : Lausanne ?
```

Comment:
Do create an issue, this should be an input field.
It could be a good first issue on Java

Reply:
Added this to the "first tasks" document.